include/seqan3/range/views/kmer_hash.hpp

```diff
@@ -174,6 +174,17 @@ class kmer_hash_view : public std::ranges::view_interface<kmer_hash_view<urng_t>
         return end();
     }
     //!\}
+
+    /*!\brief Returns the size of the range, if the underlying range is a std::ranges::sized_range.
+     * \returns Size of range.
+     */
+    uint64_t size()
```

Comment:
```suggestion
    auto size()
```

can this be const qualified?

```suggestion
    auto size() const
```